app/views/applications/_experiences_fields.html.haml

```diff
@@ -0,0 +1,20 @@
+// TODO can this div be a row? layout dillemas
+= content_tag :div, class: "nested-fields", data: { new_record: f.object.new_record? } do
+  .container.experiences_row.w-100.my-4.p-3.border.border-secondary
+    .row
+      .col-md-4
+        = f.input :project_name, label: t(".project_name")
+      .col-md-4
+        = f.input :role, label: t(".role")
+      .col-md-4
+        = f.input :budget, input_html: { min: '0',
+        required: true,  oninput: "this.value = Math.abs(this.value)",
+        placeholder: 0},
+        label: t(".budget")
+    .row
+      .col-md-6
+        = f.input :project_date, label: t(".project_date")
+      .col-md-6
+        = f.input :member_name, label: t(".member_name")
+    %small= link_to t(:delete), "#", class: "btn btn-primary", data: { action: "click->nested-form#remove_association" }
```

Comment:
Please add appropriate entry in `config/locales/pl.yml`. IIRC it should be `pl.applications.form.experiences_fields.remove_experience_item`